smart-send-logistics/block.json

```
@@ -0,0 +1,13 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 3,
+	"name": "create-block/shipping-workshop",
+	"version": "1.0.0",
+	"title": "Shipping Workshop",
+	"category": "widgets",
+	"icon": "palmtree",
+	"textdomain": "shipping-workshop",
```

This needs to match the plugin's `Text Domain` 

```suggestion
	"textdomain": "smart-send-logistics",
```